DOC-6238 Create release notes for CBL 2.6.4

ibsoln opened the pull request:
https://issues.couchbase.com/browse/DOC-6238

Amendments to Java (Android), Swift and Objc Release notes

amarantha-k commented on modules/ROOT/pages/objc.adoc:
```diff
@@ -1086,9 +1090,15 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
+
+This maintenance release includes the following important fix:
+
+- https://issues.couchbase.com/browse/CBSE-7581[CBSE-7581] Fixes memory leak, preventing crashes
```
We do not list CBSEs in the release notes. Please use the associated CBL instead for all files in the PR.

amarantha-k commented on modules/ROOT/pages/swift.adoc:
```diff
@@ -1068,9 +1069,15 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
 === 2.6.3
 
-*Fixed in this Release*
+This maintenance release includes the following important fixes:
```
Would avoid changing the format in a maintenance release. 
For 2.7.x releases, you could add a sub-heading for fixed issues (==== Fixed Issues) which is similar to the Server release notes.

ibsoln replied:
Thanks @amarantha-k . I've corrected these issues in latest commit  f94b921977b93ab711abf96dfc87bbfa34794718, which uses CBL-540 and CBL-619 in place of CBSE refs.

ibsoln committed:
https://issues.couchbase.com/browse/DOC-6238

Amendments to Java (Android), Swift and Objc Release notes

Change from CBSE to CBL ticket references
